modules/compose/compose_api_test.go

```diff
@@ -101,6 +102,42 @@ func TestDockerComposeAPIWithRunServices(t *testing.T) {
 	assert.Contains(t, serviceNames, "nginx")
 }
 
+func TestDockerComposeAPIWithStopServices(t *testing.T) {
+	compose, err := NewDockerComposeWith(
+		WithStackFiles("./testresources/docker-compose-complex.yml"),
```

Comment:
Could you use filepath to build an OS-agnostic path?

Reply:
There is also a constant for the yaml file

Reply:
Done. And I rebase onto the latest main branch.